packages/effect/test/Option.test.ts

```diff
@@ -473,16 +473,12 @@ describe("Option", () => {
   })
 
   it("all/ tuple", () => {
-    assertType<Option.Option<[number, string]>>(Option.all([Option.some(1), Option.some("hello")]))
```

Comment:
Seems like `Option.tst.ts` already has assertions equivalent to the ones removed.